router/core/router.go

```diff
@@ -703,14 +706,26 @@ func (r *Router) newServer(ctx context.Context, routerConfig *nodev1.RouterConfi
 		Config:            r.Config,
 	}
 
+	baseAttributes := []attribute.KeyValue{
+		otel.WgRouterConfigVersion.String(routerConfig.GetVersion()),
+		otel.WgRouterVersion.String(Version),
+	}
+
+	if r.graphApiToken != "" {
+		claims, err := jwt2.ExtractFederatedGraphTokenClaims(r.graphApiToken)
```

Comment:
Please use `rjwt` if we have a collision with the router packages.

Reply:
Done